app/helpers/dashboard_helper.rb

```diff
@@ -44,6 +44,6 @@ def percentage_string(percentage)
   def percentage(numerator, denominator)
     return percentage_string(0) if denominator.zero?
 
-    percentage_string((numerator * 100) / denominator)
+    percentage_string((numerator * 100.0) / denominator)
```

Comment:
Now that this is a floating point number, do we need to specify a precision to ensure we don't generate strings like "8.33333333%"?

Reply:
The `percentage_string` function has a `.round(0)` that will handle this. It was getting an integer input earlier 🤦‍♂

Reply:
Understood. Was the "off-by-1-percent" a real issue?